spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -47,19 +47,163 @@ definitions:
+      a space vehicle
+    fields:
+    - satId:
+        type: u8
+        desc: ID of the space vehicle within its constellation
+    - constellation:
+        type: u8
+        desc: Constellation ID to which the SV belongs
+
+ - STECHeader:
+    short_desc: Header for MSG_SSR_STEC_CORRECTION message
+    desc: |
+      A full set of STEC information will likely span multiple SBP
+      messages, since SBP message a limited to 255 bytes.  The header
+      is used to tie multiple SBP messages into a sequence.
+    fields:
+    - time:
+        type: GPSTime
+        desc: GNSS time of the STEC data
+    - num_msgs:
+        type: u8
+        desc: Number of messages in the dataset
+    - seq_num:
+        type: u8
+        desc: Postion of this message in the dataset
+    - ssr_update_interval:
+        type: u16
+        desc: update interval in seconds
+    - iod_ssr:
+        type: u8
+        desc: range 0 - 15
+
+ - GriddedCorrectionHeader:
+    short_desc: Header for MSG_SSR_GRIDDED_CORRECTION
+    desc: |
+      The 3GPP message contains nested variable length arrays
+      which are not suppported in SBP, so each grid point will
+      be identified by the index.
+    fields:
+    - time:
+        type: GPSTime
+        desc: GNSS time of the STEC data
+    - num_msgs:
+        type: u8
+        desc: Number of messages in the dataset
+    - seq_num:
+        type: u8
+        desc: Postion of this message in the dataset
+    - ssr_update_interval:
+        type: u16
+        desc: update interval in seconds
+    - iod_ssr:
+        type: u8
+        desc: range 0 - 15
+    - tropo_quality:
+        type: u8
+        desc: troposphere quality indicator
+    - index:
+        type: u8
+        desc: index of the grid point
+
+ - STECSatElement:
+     short_desc: SSR code biases corrections for a particular satellite.
+     desc: |
+       Code biases are to be added to pseudorange.
+       The corrections are conform with typical RTCMv3 MT1059 and 1065.
+     fields:
+     - sv_id:
+         type: SvId
+         desc: Unique space vehicle identifier
+     - stec_quality_indicator:
+         type: u8
+         desc: quality of STEC data
+     - stec_coeff:
+         type: array
+         size: 4
+         fill: s16
+         desc: coefficents of the STEC polynomial
+
+ - TroposphericDelayCorrection:
+    short_desc: troposphere delay correction
+    desc: |
+      Contains wet vertical and hydrostatic vertical delay
+    fields:
+    - hydro:
+        type: s16
+        desc: hydrostatic vertical delay
+    - wet:
+        type: s8
+        desc: wet vertical delay
+
+ - STECResidual:
+    desc: STEC residual
+    fields:
+    - sv_id:
+        type: SvId
+        desc: space vehicle identifier
+    - residual:
+        type: s16
+        desc: STEC residual
+
+ - GridElement:
+    short_desc: Grid datum for troposphere and STEC residuals
+    desc: |
+      Contains one tropo datum, plus STEC residuals for each space
```

Comment:
I guess we are trying to keep close to the 3GPP proposals, but combining tropo and iono does mean we can't have different update rates for the two types of data if there was a desire to reduce data rate

Reply:
dunno... above my pay grade here :)  @anth-cole @benjamin0 ?

Reply:
Good point, @ljbade. I'm in favor of sticking with the Compact SSR messages for now, but we can always split them out later.